BatteryStatus.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+
+namespace BatteryManagement
+{
```

Comment:
Class name  "BatteryStatus" doesn't justify for the encapsulated functionalities
Separate Core Logic Functionalities from Display functions

Reply:
Name changed and separated core logic and dislpay messages